.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+<!--
+List the issues this PR closes (if any) in a bullet list format, e.g.:
+- closes https://github.com/FuelLabs/fuels-ts/issues/ABCD
+- closes https://github.com/FuelLabs/fuels-ts/issues/EFGH
+-->
+
+# Summary
+
+<!--
+Please write a summary of your changes and why you made them.
+Not all PRs will be complex or substantial enough to require this section, so you can remove it if you think it's unnecessary.
+-->
+
+# Breaking changes
+
+<!--
+If the PR has breaking changes, please detail them in this section between the START and END comments below.
+-->
+
+<!--START: Breaking changes section. Used in automation so do not remove this comment. -->
```

Comment:
Can the section, including the comments, be removed if there are no breaking changes?

Reply:
Yes, the migration note changelog logic only searches for this section on breaking change PRs.

Reply:
Do we actually need the comments?

Couldn't we get everything between `# Breaking changes` and `# Checklist`?

Reply:
You're right: https://github.com/FuelLabs/fuels-ts/pull/2688/commits/39e31237180fb09fe0c2e1101c875e82e235d93e.